src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -37,6 +37,113 @@ interface PharmacyProps {
   lastUpdated: string;
   vaccineAvailabilities: VaccineAvailabilitiesByDateAndRequirementsInterface;
 }
+export const dateMarkup = (createdAtDateString: string) => {
+  const createdAtDate = new Date(createdAtDateString);
+  const currentDate = new Date();
+  const secondsSinceCreated = Math.floor(
+    (currentDate.getTime() - createdAtDate.getTime()) / 1000,
+  );
+  const months = [
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "June",
+    "July",
+    "Aug",
+    "Sept",
+    "Oct",
+    "Nov",
+    "Dec",
+  ];
+  if (secondsSinceCreated < 60) {
+    return "Just Now";
+  }
+  if (secondsSinceCreated > 60 && secondsSinceCreated < 3600) {
```

Comment:
We really should be leveraging `date-fns` when we're working with dates... working with dates directly like this is cumbersome and error prone.

For all these comparisons, `isAfter()`, `isBefore`, `addSeconds`, `addMinutes` etc would result in a simpler implementation.